kp_service_test/src/test/resources/templates/content/v3/update/testUpdateContentNewFramework/validate.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "ekstep.content.find",
+  "ver": "3.0",
+  "ts": "@ignore@",
+  "params": {
+    "resmsgid": "@ignore@",
+    "msgid": null,
+    "err": null,
+    "status": "successful",
+    "errmsg": null
+  },
+  "responseCode": "OK",
+  "result": {
+    "content": {
+      "ownershipType": [
+        "createdBy"
+      ],
+      "code": "kp.ft.resource.pdf",
+      "channel": "channel-01",
+      "description": "KP Integration Test Content",
+      "language": [
+        "English"
+      ],
+      "mimeType": "application/pdf",
+      "idealScreenSize": "normal",
+      "createdOn": "@ignore@",
+      "appId": "@ignore@",
+      "contentDisposition": "inline",
+      "contentEncoding": "identity",
+      "lastUpdatedOn": "@ignore@",
+      "contentType": "Resource",
+      "dialcodeRequired": "No",
+      "identifier": "@ignore@",
+      "lastStatusChangedOn": "@ignore@",
```

Comment:
please use @matchesDatePattern('yyyy-MM-dd')@

Reply:
Fixed this